### PR TITLE
Use book class to structure thesis in chapters

### PR DESCRIPTION
**Pull request by petobens**

### CODE REVIEW

**Comment by petobens** [comment]
Note: uncomment algorithm package and environment. Check if you can create a bibibliography section using biber.

**Comment by jdemonasterio** [comment]
It is now linux compiling. Will approve PR request and ammend the dirty `biber` fix where blank pages are inserted among sections. :raised_hands:

File: preamble.tex [hunk 1]
```diff
@@ -14,7 +14,7 @@
 %--------------------------------------+
 % Language, hyphenation, encoding, etc |
 %--------------------------------------+
-\usepackage[spanish,english]{babel}
+\usepackage[spanish,english]{biblatex}
```
Comment (petobens) [inline preamble.tex]: This seems to be wrong.

File: preamble.tex [hunk 2]
```diff
@@ -159,8 +159,21 @@
...
-\usepackage[style=authoryear-comp,backref=true,refsection=chapter,
-backend=biber]{biblatex}
+%\usepackage[style=authoryear-comp,backref=true,refsection=chapter,
+%backend=biber]{biblatex}
+
+% this might insert blank pages here and there but now compilation works correctly with biber
+\usepackage[
+	backend=biber,
+	sortlocale=us_EN,
+	natbib=true,
```
Comment (petobens) [inline preamble.tex:169]: Natbib option doesn't affect the authoryear-comp style AFAIR, therefore this settings is redundant.

File: preamble.tex (hunk 1 above)
Comment (jdemonasterio) [reply preamble.tex]: Yes this was not included in the `real` commit. That line was directly commented. 

p.s.: the `real` part comes from the inadvertent choice of githubs' complicated fork--> PR --> review PR --> approve PR .. process. I had to manually edit files online in the third step, missing this edit that was done locally.

if you know better how to commit to an open Fork PR without push access, be welcomed :)

File: preamble.tex (hunk 2 above)
Comment (jdemonasterio) [reply preamble.tex:169]: Great thx. All changes are being made in the main repository's master branch.

**Pull request merged**